tests/ha/iscsi_client_setup.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: iscsi-client-sles-16
+# Summary: Configure iSCSI target for HA tests
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use Utils::Backends qw(is_remote_backend);
+use utils qw(zypper_call systemctl ping_size_check);
+use testapi;
+use hacluster;
+use version_utils qw(is_sle package_version_cmp);
+
+sub run {
+    my $iscsi_server = get_var('USE_SUPPORT_SERVER') ? 'ns' : get_required_var('ISCSI_SERVER');
+
+    # Perform a ping size check to several hosts which need to be accessible while
+    # running this module
+    ping_size_check(testapi::host_ip());
+    ping_size_check($iscsi_server);
+
+    # open-iscsi & iscsiuio
+    if (script_run('rpm -q open-iscsi') != 0) {
+        zypper_call 'in open-iscsi';
+    }
+
+    record_info('iscsi_client version', script_output('rpm -q open-iscsi'));
+    assert_script_run("systemctl start iscsid");
+    my $iscsi_daemon_status = script_output("systemctl status iscsid");
+    record_info('iscsi_client version', $iscsi_daemon_status);
+
+    my $iqn_node_name = script_output("iscsiadm -m discovery -t st -p 10.0.2.1|grep 'openqa'");
```

Comment:
Hardcoded 10.0.2.1 address. Makes it fail on `pvm_hmc` backend: https://openqa.suse.de/tests/17108044#step/iscsi_client_setup/43

Please fix.